web/src/components/tasks/business-structure/BusinessStructureTask.tsx

```diff
@@ -68,6 +68,7 @@ export const BusinessStructureTask = (props: Props): ReactElement => {
 
     queueUpdateTaskProgress(props.task.id, "COMPLETED");
     await updateQueue.queueProfileData(profileData).update();
+    await updateQueue.queueFormationFormData(business.formationData.formationFormData).update();
```

Comment:
pebble - what is the purpose of this addition? And where is the accompanying test?

Reply:
Removed and implemented logic in `userRouter.ts` instead, because that is where the data is being reset (clearing the address fields) by way of `formationFormData: createEmptyFormationFormData()`.